kong/db/migrations/core/013_220_to_230.lua

```diff
@@ -0,0 +1,48 @@
+local utils = require("kong.tools.utils")
+
+
+return {
+  postgres = {
+    up = [[
+      CREATE TABLE IF NOT EXISTS "parameters" (
+        key            TEXT PRIMARY KEY,
+        value          TEXT NOT NULL,
+        created_at     TIMESTAMP WITH TIME ZONE
+      );
+    ]],
+    teardown = function(connector)
+      local _, err = connector:query(string.format([[
+        INSERT INTO parameters (key, value) VALUES('%s', '%s')
+        ON CONFLICT DO NOTHING;
+      ]], "cluster_id", utils.uuid()))
```

Comment:
Since this code runs in `teardown` phase, there can _potentially_ be a time during which `kong.cluster.get_id()` is called in a Kong 2.3 node, but the ID is not present in the database (nodes of the newer versions are allowed to startup even when `kong migrations finish` has not bee run). Can that create a potential problem?

Reply:
Yes, the issue was that for some reason only the `teardown` handler actually supports a function. But now I rethink about it, we could generate a `up` SQL that contains a randomly generated UUID in it. I will make that change.

Reply:
Fixed.